internal/filter/filterlog/filterlog_test.go

```diff
@@ -9,12 +9,16 @@ import (
 
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+	"go.opentelemetry.io/collector/featuregate"
 	"go.opentelemetry.io/collector/pdata/pcommon"
 	"go.opentelemetry.io/collector/pdata/plog"
+	conventions "go.opentelemetry.io/collector/semconv/v1.6.1"
```

Comment:
I believe there is a more recent version of this

Reply:
I agree, but internal/filter already has precedence for these versions, so I'd rather not update them as part of this PR.